src/celeritas/ext/detail/HitProcessor.cc

```diff
@@ -249,7 +290,7 @@ PrimaryId HitProcessor::register_primary(G4Track const& primary)
  */
 void HitProcessor::begin_event()
 {
-    celeritas_to_g4_track_id_.clear();
+    celeritas_to_g4_track_data_.clear();
```

Comment:
Should we try to clear user track information before the end of the previous event? Are there any user info classes that have destructors with side effects that assume the event is the same?

Reply:
There's also the auxiliary track information that are deleted when the track is destroyed